src/main/java/items/Items.java

```diff
@@ -0,0 +1,112 @@
+package items;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import handler.DataHandler;
+import msg.StdMsgs;
+import task.Task;
+import msg.Msg;
+
+
+/**
+ * The items class is a representation of a list of tasks that is able to add, delete, mark and unmark tasks on that
+ * list and return an appropriate response
+ */
+public class Items {
+    /** tracks number of tasks created*/
+    private int item_count = 0;
+    /** ArrayList<task> to store tasks */
+    private ArrayList<Task> item_list = new ArrayList<Task>();
+
+    public Items() {}
+
+    /**
+     * Adds an item to item_list and gives appropriate response msg
+     *
+     * @param item
+     */
+    public void add(Task item) throws IOException {
```

Comment:
I like how you have made sure to put a space before the opening '{' brace, as well as in other appropriate places in the method signature. Have also noticed that you have done this for other methods.